### ISSUE TITLE
Refer to publicly accessible jquery-cloudbrowser

### ISSUE THREAD

**Issue by matthewlmcclure** [brianmcd/cloudbrowser#43]
The [.gitmodules](https://github.com/brianmcd/cloudbrowser/blob/master/.gitmodules#L3) file refers to `url = git@theta.cs.vt.edu:jquery-cloudbrowser.git`, which breaks `make setup` for people without access to `theta.cs.vt.edu`.

```
$ make setup
git submodule update --init
Submodule 'deps/jquery-cloudbrowser' (git@theta.cs.vt.edu:jquery-cloudbrowser.git) registered for path 'deps/jquery-cloudbrowser'
Initialized empty Git repository in /vagrant/cloudbrowser/deps/jquery-cloudbrowser/.git/
The authenticity of host 'theta.cs.vt.edu (128.173.237.152)' can't be established.
RSA key fingerprint is 86:35:2e:1f:c6:f9:93:f7:d2:7c:3e:17:ea:c1:c2:29.
Are you sure you want to continue connecting (yes/no)? yes

Warning: Permanently added 'theta.cs.vt.edu,128.173.237.152' (RSA) to the list of known hosts.
git@theta.cs.vt.edu's password:   C-c
make: *** wait: No child processes.  Stop.
make: *** Waiting for unfinished jobs....
make: *** wait: No child processes.  Stop.
```

**Comment by brianmcd** [comment]
Good catch. I actually don't have access to theta anymore either - maybe @godmar could host that repository somewhere on github and we could update .gitmodules?  FWIW, jquery-cloudbrowser is only used for testing purposes, so everything else _should_ work without it.

It might make sense to move cloudbrowser to its own Github organization which hosts all of its child repositories.

**Comment by godmar** [comment]
I thought that 'make setup' is obsolete anyway, we're not using git submodules anymore. 'npm install' is all that's needed.

Brian, you should still have access to theta - did you update/change your ssh key? If so, I'll install your new key.
Currently, you have 4 ssh keys in there.

Right now, the leading edge of cloudbrowser development is on a branch called 'deployment'. I've pushed that branch to the github repo, see https://github.com/brianmcd/cloudbrowser/tree/deployment Ashima Athri has been doing a fair amount of development there, with the goal of implementing a hosting platform for application deployment. She's also implemented local & Google (OAuth2)-based authentication.

Please use that branch to report problems, if possible. We'll eventually merge that branch back into master (provided the git gods are good to us.)

Re: child repositories: we don't have any anymore. cloudbrowser-jquery is, as you say, only needed for testing. cloudbrowser-jsdom is no longer needed since we npm install the actual jsdom, then patch it. I'm currently not sure about the html5 & knockout, but those point in the package.json to github anyway.  jsdom is the really important one here.
